docs/docsite/rst/dev_guide/developing_program_flow_modules.rst

```diff
@@ -185,83 +175,85 @@ Executor/task_executor
 
 The TaskExecutor receives the module name and parameters that were parsed from
 the :term:`playbook <playbooks>` (or from the command line in the case of
-:command:`/usr/bin/ansible`).  It uses the name to decide whether it's looking
-at a module or an :ref:`Action Plugin <flow_action_plugins>`.  If it's
+:command:`/usr/bin/ansible`). It uses the name to decide whether it's looking
+at a module or an :ref:`Action Plugin <flow_action_plugins>`. If it's
 a module, it loads the :ref:`Normal Action Plugin <flow_normal_action_plugin>`
 and passes the name, variables, and other information about the task and play
 to that Action Plugin for further processing.
 
 .. _flow_normal_action_plugin:
 
-Normal action plugin
---------------------
+The ``normal`` action plugin
+----------------------------
 
-The ``normal`` action plugin executes the module on the remote host.  It is
+The ``normal`` action plugin executes the module on the remote host. It is
 the primary coordinator of much of the work to actually execute the module on
 the managed machine.
 
-* It takes care of creating a connection to the managed machine by
+* It creates a connection to the managed machine by
```

Comment:
It uses a connection plugin to create a connection, mostly indirectly by asking the connection plugin to transfer the module or other files ... it does not really create it itself directly.